pkg/pillar/hypervisor/kvm.go

```diff
@@ -707,6 +726,44 @@ func vmmOverhead(domainName string, domainUUID uuid.UUID, domainRAMSize int64, v
 	return overhead, nil
 }
 
+func getOVMFSettingsFile(domainName string) string {
+	return types.OVMFSettingsDir + "/" + domainName + "_OVMF_VARS.fd"
+}
+
+func prepareOVMFSettings(domainName string) (string, error) {
+	logrus.Infof("@ohm: I Preparing OVMF settings for domain %s", domainName)
+
+	// Create the OVMF settings directory if it does not exist
+	if _, err := os.Stat(types.OVMFSettingsDir); os.IsNotExist(err) {
+		if err := os.MkdirAll(types.OVMFSettingsDir, 0755); err != nil {
+			return "", logError("failed to create OVMF settings directory: %v", err)
+		}
+	}
+	// Create a copy of the OVMF_VARS.fd file in <UUID>_OVMF_VARS.fd
+	ovmfVarsFile := getOVMFSettingsFile(domainName)
+	if _, err := os.Stat(ovmfVarsFile); os.IsNotExist(err) {
+		// Copy the OVMF_VARS.fd file, using statndard copy function
```

Comment:
```suggestion
		// Copy the ovmf_vars.bin file, using standard copy function
```

Reply:
Yeah, I have a mess in these terms... Will have to clean it up later.

Reply:
Removed the entire comment, it was just for me